src/interfaces.ts

```diff
@@ -152,6 +152,12 @@ export function entityToFieldName(entity: IChainEntityKind): string | null {
     case MolochTypes.EntityKind.Proposal: {
       return 'proposalIndex';
     }
+    case MarlinTypes.EntityKind.Proposal: {
+      return 'proposalIndex';
+    }
+    case MarlinTypes.EntityKind.Receipt: {
```

Comment:
Do we ever use this entity type? I don't see anything with a "receiptIndex".

Reply:
We don't use this, so i'll just remove it